v2/prixite/pages/index.js

```diff
@@ -0,0 +1,230 @@
+import {
+  Button,
+  Container,
+  Divider,
+  Grid,
+  Stack,
+  Typography,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Head from "next/head";
+import Image from "next/image";
+import Footer from "../components/footer";
+import Header from "../components/header";
+import homePageBg from "../public/images/homePageBg.png";
+import devOpsAsService from "../public/images/devOpsAsService.png";
+import webDevelopment from "../public/images/webDevelopment.jpg";
+import mobileDev from "../public/images/mobileDev.jpg";
+import scraping from "../public/images/scraping.jpg";
+import pwa from "../public/images/pwa.jpg";
+import ArrowRightAltIcon from "@mui/icons-material/ArrowRightAlt";
+import reactjs from "../public/images/reactjs.jpg";
+import wordpress from "../public/images/wordpress.jpg";
+import python from "../public/images/python.jpg";
+
+export default function Home() {
```

Comment:
This should be Index?

Reply:
Name: "index" for a component would be confusing,, I will make it better in other way